submissions/HelenGreent/js-dom/index.js

```diff
@@ -0,0 +1,73 @@
+  },
+  {
+    name: "Hyoscyamus niger",
+    blossoming: "June-August",
+    location: "Europe and Siberia, Great Britain, Ireland",
+    image: "https://herbalapothecaryuk.com/wp-content/uploads/2020/07/3086_fresh.jpg",
+    desc: "Hyoscyamus niger, commonly known as henbane, black henbane, or stinking nightshade, is a plant that is poisonous in large quantities, in the nightshade family Solanaceae. It is native to temperate Europe and Siberia, and naturalised in Great Britain and Ireland. Henbane ingestion by humans is followed simultaneously by peripheral inhibition and central stimulation.[14] Common effects of henbane ingestion include hallucinations,[2] dilated pupils, narcosis, restlessness, and flushed skin. Less common effects are tachycardia, convulsions, vomiting, hypertension, hyperpyrexia, and ataxia. Initial effects typically last for three to four hours, while aftereffects may last up to three days. The side effects of henbane ingestion are dryness in the mouth, confusion, visual illusions, bizarre thoughts, locomotor and memory disturbances, and farsightedness, similar in style to those of other tropane-based deliriants such as plants of the New World genus datura. As a result of this distinct chemical and pharmacological profile, overdoses can result not only in delirium, but also severe anticholinergic syndrome, coma, respiratory paralysis, and death. Low and average dosages have inebriating and aphrodisiac effects.",
+  },
+];
+
+const main = document.querySelector("#main");
+const nav = document.querySelector("#nav");
+
+let navLinks = herbs.map((herb) => `<li class ='nav_item'><a href='#'>${herb.name}</a></li>`).join("");
+nav.innerHTML = navLinks;
+
+function createArticle(item) {
+  const article = document.createElement("article");
+  article.className = "article";
+  article.innerHTML = `
+    <div class='article-wrapper'>
+    <div class = 'article-desc'>
+        <div><h2>Name: </h3><span> ${item.name}</span></div>
+        <div><h2>Blossom time: </h3><span> ${item.blossoming}</span></div>
+        <div><h2>Location: </h3><span> ${item.location}</span></div>
+    </div>
+    <figure>
+        <img class='image' src='${item.image}' alt='${item.name}'/>
+        <figcaption class='figcaption'>${item.name}</figcaption>
+    </figure>
+    </div>
+    <p class='description'>${item.desc}</p>`;
+  return article;
+}
+
+nav.addEventListener("click", (event) => {
+  let currentLink;
```

Comment:
What is happening here? `currentLink` will always be `undefined` and the condition below will always work, since `currentLink` is created each time you click a button and immediately gets deleted since it is not used outside this function. Maybe you wanted to prevent the user from reclicking the same button, but it didn't work. You should move the `currentLink` outside this function so it can store a value. Also, if you want something to be `empty`, use `null` value. `undefined` usually means someone forgot to place a value or something is lost.